comps/cores/mega/gateway.py

```diff
@@ -156,9 +156,12 @@ def __init__(self, megaservice, host="0.0.0.0", port=8888):
 
     async def handle_request(self, request: Request):
         data = await request.json()
+        print("data in handle request", data)
```

Comment:
If the `print` statements are for debugging purpose, I'd remove them.

Reply:
I just added GraphRagGateway to the end of the file and this belongs to chatqna gateway. maybe they kept those for a reason, I wont touch.